getting_started/step_by_step/ui_code_a_life_bar.rst

```diff
@@ -171,12 +171,12 @@ function and look at the health value? Accessing nodes this way creates
 tight coupling between them. If you did it sparingly it may work. As
 your game grows bigger, you may have many more connections. If you get
 nodes from a bad it's becomes very complex very soon. Not only that: you
```

Comment:
`from a bad`?

Reply:
No idea, IMO just replace it with "If you get nodes from a bad it becomes..." - Also, notice the typo on "it's", should be "it"

Reply:
Please note the next time when you submitted the change yourself :smile: 
Was about to submit a PR...